contracts/stellar-interchain-token-service/src/token_metadata.rs

```diff
@@ -42,11 +46,31 @@ impl TokenMetadataExt for TokenMetadata {
             !self.symbol.is_empty() && self.symbol.len() <= MAX_SYMBOL_LENGTH,
             ContractError::InvalidTokenSymbol
         );
+        ensure!(is_ascii_string(&self.name), ContractError::InvalidTokenName);
+        ensure!(
+            is_ascii_string(&self.symbol),
+            ContractError::InvalidTokenSymbol
+        );
 
         Ok(())
     }
 }
 
+fn is_ascii_string(input: &String) -> bool {
```

Comment:
Create a `StringExt` trait that defines an `is_ascii` method in `stellar-axelar-std` to do this in a cleaner way